descriptors/ModuleDescriptor-template.json

```diff
@@ -933,15 +933,15 @@
     },
     {
       "id" : "feefine-reports",
-      "version" : "1.0",
+      "version" : "2.0",
       "handlers": [
         {
           "methods":[
-            "GET"
+            "POST"
```

Comment:
yes, it was done to avoid URL characters restrictions. We decided to not limit ownerIds in case there are more then 20 of them on production environment. These changes were discussed with FE team and UI part will support POST

Reply:
Ok, if you think it's necessary to use POST then this change should be kept. Sometimes we programmers account for all cases that might never be used, and it's better to avoid changing the API drastically as much as possible, especially in this case when we have to use POST to do a GET.  IMHO even if they would send in 40 UUIDs there would still be room for it, and 20 is a practical maximum, so a GET is good enough, but since there were prior discussions I trust your judgment.